zuchini-model/src/main/java/org/zuchini/parser/FeatureParser.java

```diff
@@ -51,7 +51,15 @@ public static Feature getFeature(Reader reader) throws IOException {
     }
 
     public static Feature getFeature(String input) {
-        return getFeature(new ANTLRInputStream(input));
+        return getFeature(new ANTLRInputStream(appendNewLineAtEOFIfMissing(input)));
+    }
+
+    private static String appendNewLineAtEOFIfMissing(String input) {
+        final String newLine = System.getProperty("line.separator");
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--

Reply:
you mean I should hardcode `\n` instead of using a system dependent line separator?